app/src/hog.c

```diff
@@ -156,28 +157,120 @@ struct bt_conn *destination_connection() {
     return conn;
 }
 
+#define HOG_STACK_SIZE 512
+#define HOG_PRIORITY 5
```

Comment:
Does priority 5 have any particular meaning or is it arbitrary?

Reply:
See https://docs.zephyrproject.org/2.4.0/reference/kernel/scheduling/index.html#scheduling-algorithm

Thread priorities impact the scheduling algorithm. Moved to Kconfig, so we can easily tweak as we see fit.